packages/sdks/mitosis.config.cjs

```diff
@@ -718,6 +718,55 @@ const ANGULAR_COMPONENT_REF_UPDATE_TEMPLATE_SSR = () => ({
   },
 });
 
+const ANGULAR_MARK_SAFE_INNER_HTML = () => ({
+  code: {
+    post: (code) => {
+      if (
+        code.includes('selector: "builder-text"') ||
+        code.includes('selector: "builder-embed"') ||
+        code.includes('selector: "custom-code"')
+      ) {
+        code =
+          `import { DomSanitizer } from "@angular/platform-browser";\nimport { ChangeDetectionStrategy } from "@angular/core";\n` +
+          code;
+
+        // add changeDetection: ChangeDetectionStrategy.OnPush
+        const changeDetectionIndex = code.indexOf('selector:');
+        if (changeDetectionIndex !== -1) {
+          code =
+            code.slice(0, changeDetectionIndex) +
+            `changeDetection: ChangeDetectionStrategy.OnPush,\n` +
+            code.slice(changeDetectionIndex);
+        }
+
+        // add constructor with sanitizer
+        const constructorIndex = code.indexOf('constructor');
+        if (constructorIndex === -1) {
+          // not found
+          const ngOnChangesIndex = code.indexOf('ngOnChanges');
+          code =
+            code.slice(0, ngOnChangesIndex) +
+            `constructor(protected sanitizer: DomSanitizer) {}\n` +
+            code.slice(ngOnChangesIndex);
+        } else {
+          throw new Error(
+            'constructor found which should not be here. If you see this, please fix the ANGULAR_TEXT_MARK_SAFE_HTML Plugin.'
+          );
+        }
+
+        const variableName = code.match(/\[innerHTML\]="([^"]+)"/)?.[1];
+        if (variableName) {
+          code = code.replace(
+            `[innerHTML]="${variableName}"`,
+            `[innerHTML]="sanitizer.bypassSecurityTrustHtml(${variableName})"`
```

Comment:
@samijaber this was needed for Embed and Custom Code block as well so I extended this PR to fix them as well

Reply:
long-term I think this should be baked into mitosis: the angular generator should always use `sanitizer.bypassSecurityTrustHtml` for `innerHTML` (maybe with a flag to force-disable the sanitizer if needed)

Reply:
great idea, i've created a ticket for it